### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by ghost**
cc @diasdavid @jbenet @whyrusleeping @kyledrake @mappum @richardlitt

First request is #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
-Requests about infrastructure operations
+
+Use this repo's issue tracker to direct requests to the infrastructure team (mainly @lgierth).
+
+Make sure that:
+
+- the request is specific
+- the request is actionable
+- the request is prioritizable
+  - e.g. has a specific timeframe
```
Comment (daviddias) [inline README.md]: Could you add a description to the README of what the labels(https://github.com/ipfs/ops-requests/labels) mean? Urgent is Urgent, but the others might not mean the same thing to everyone.

Comment (ghost) [inline README.md]: Updated everything a bit -- I think it might actually be wiser to go without issue labels for now

**Comment by ghost** [comment]
@RichardLitt yep updated! I made ipfs.io and webchat.freenode.net use https

**Comment by ghost** [comment]
and omited the contribute gif, it's just too much blinking for reading the text below :)

**Comment by RichardLitt** [comment]
Haha aw. You make me sad.

**Pull request merged**